pkgs/development/python-modules/torch/bin.nix

```diff
@@ -109,7 +109,7 @@ in buildPythonPackage {
     # torch-bin used to vendor CUDA. It still links against CUDA and MKL.
     license = with licenses; [ bsd3 issl unfreeRedistributable ];
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    platforms = [ "aarch64-darwin" "aarch64-linux" "x86_64-darwin" "x86_64-linux" ];
```

Comment:
`x86_64-darwin` has simply been discontinued.
See the _Deprecations/Releng_ section of [the changelog](https://github.com/pytorch/pytorch/releases/tag/v2.3.0).

Reply:
Wow, fair enough, seems harsh, but it's upstream's decision https://github.com/pytorch/pytorch/issues/114602

Reply:
It seems it can still theoretically be built for Intel Macs? Quite a bit of resistance to upstream's decision, we could still package it https://github.com/pytorch/pytorch/issues/114602#issuecomment-1957213622